R/knitr.R

```diff
@@ -0,0 +1,54 @@
+#' Register the knitr engine for Stan
+#'
+#' Registers CmdStanR's knitr engine [eng_stan()] for processing Stan chunks.
```

Comment:
Can you mention in the doc that there's a vignette with an example?

Reply:
Sure thing!